setup.sh

```diff
@@ -12,11 +14,10 @@ chmod +x .git/hooks/pre-push
 echo "${G}Pre-Push hooks set.${NC}"
 
 echo "${Y}Installing deps...${NC}"
-poetry shell
-poetry install
+poetry run install
```

Comment:
I think this one should stay as `poetry install`, otherwise `poetry run` will try to execute the `install` program (typically at `/usr/bin/install`).

Reply:
I did run it and it seemed alright 🤔 maybe cause I had an already set env?

Reply:
I meant that it will run but won't be doing what you expect.

```
$ poetry run install
usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
```

I think `poetry install` fits the intention ("Installing deps...").